src/network/networkprotocol.h

```diff
@@ -602,14 +605,22 @@ enum ToClientCommand
 
 	TOCLIENT_SET_SKY = 0x4f,
 	/*
-		u8[4] color (ARGB)
-		u8 len
-		u8[len] type
-		u16 count
-		foreach count:
-			u8 len
-			u8[len] param
-		u8 clouds (boolean)
```

Comment:
The old format documentation should be kept until the compatibility code is dropped.